examples/examples/get_blocks.rs

```diff
@@ -27,6 +28,7 @@ use substrate_api_client::{
 #[tokio::main]
 async fn main() {
```

Comment:
What do you think: Do we still need the sync variant of get_blocks? We have different sync examples and the difference between sync and async is not that big.

Reply:
Initially I thought it would be nice to show that the async version is faster. Unfortunately that's not really the case. The sync has the small advantage of outputting something directly after each getter, so if something fails later on there is at least the output of the previous stuff.